webapp/src/Entity/JudgingRunOutput.php

```diff
@@ -116,6 +123,17 @@ public function getOutputSystem(): string
         return $this->output_system;
     }
 
+    public function setTeamMessage(string $teamMessage) : JudgingRunOutput
+    {
+        $this->team_message = $teamMessage;
+        return $this;
+    }
+
+    public function getTeamMessage(): string
```

Comment:
This might need
```suggestion
    public function getTeamMessage(): ?string
```
to work?